Submodels/Parameterizations.py

```
@@ -0,0 +1,775 @@
+        self.idx_maxT = np.where(self.thickness_distribution == np.max(self.thickness_distribution))[0][0]
+        self.idx_maxC = np.where(self.camber_distribution == np.max(self.camber_distribution))[0][0]
+        
+        self.x_t = self.x_points_thickness[self.idx_maxT]  # X-coordinate of maximum thickness
+        self.y_t = self.thickness_distribution[self.idx_maxT]  # Maximum thickness
+        self.x_c = self.x_points_camber[self.idx_maxC]  # X-coordinate of maximum camber
+        self.y_c = self.camber_distribution[self.idx_maxC]  # Maximum camber
+        self.z_TE = self.camber_distribution[-1]  # Trailing edge vertical displacement
+        self.dz_TE = self.thickness_distribution[-1]  # Half thickness of the trailing edge
+
+        # To find the radius of curvature of the leading edge, we fit a circle to the points within the first 2 % of the upper surface      
+        def GetLeadingEdgeRadius(params, x, y):
```

Ruff (0.8.2)

Missing return type annotation for private function `GetLeadingEdgeRadius`

(ANN202)